internal/ingest/interface.go

```diff
@@ -9,11 +9,11 @@ import (
 
 // Ingester is the interface implemented to subscribe to
 // advertisements from a provider
-type Ingester interface {
+type XIngester interface {
```

Comment:
is the X here intentional?

Reply:
Meant to delete that interface.  Done

Reply:
The X meant it was marked for removal... just need to delete it. Done.